### PR TITLE
lectia-1: add responsive box styles

### PR DESCRIPTION
**Pull request by alexcracea**

### CODE REVIEW

File: scss/components/_responisve-box.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.responsive-box {
+  width: 400px;
+  height: 400px;
```
Comment (alexcracea) [inline scss/components/_responisve-box.scss:3]: Please add more height

File: scss/components/_responisve-box.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+.responsive-box--2 {
+  color: black;
+}
+
+.responsive-box--3 {
+  color: black;
+}
+
```
Comment (DorinLisii) [inline scss/components/_responisve-box.scss:17]: no comment

**Review by DorinLisii — APPROVED** [review]

File: scss/components/_responisve-box.scss (hunk 1 above)
Comment (DinuVidrascu) [inline scss/components/_responisve-box.scss:3]: good

File: scss/components/_responisve-box.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
+.responsive-box {
```
Comment (Anghel100500) [inline scss/components/_responisve-box.scss:1]: is de acord total

File: scss/components/_responisve-box.scss (hunk 1 above)
Comment (anaspatari) [reply scss/components/_responisve-box.scss:3]: approved

**Review by Anghel100500 — APPROVED** [review]
cam putin)

**Review by anaspatari — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# css_advanced
```
Comment (JanaBulmaga) [inline README.md:1]: ac

**Review by JanaBulmaga — APPROVED** [review]

**Pull request merged**